app/people/customer.py

```diff
@@ -0,0 +1,7 @@
+class Customer:
+    def __init__(self, name: str, food: str) -> None:
+        self.name = name
+        self.food = food
+
+    def watch_movie(self, movie: str) -> None:
+        print(f"{self.name} is watching \"{movie}\".")

```

Comment:
```suggestion
    def watch_movie(self, movie: str) -> None:
        print(f'{self.name} is watching "{movie}".')
```
try this, it shouldn't cause the flake8 error